ksp_plugin_adapter/main_window.cs

```diff
@@ -367,13 +364,13 @@ internal class MainWindow : SupervisedWindowRenderer {
           value   : must_record_journal_,
           text    : "Record journal (starts on load)");
       UnityEngine.GUILayout.Label(
-          "Journalling is " + (journaling_ ? "ON" : "OFF"),
+          "Journalling is " + (journalling_ ? "ON" : "OFF"),
```

Comment:
It seems the spelling of the variable was good and that the string should be changed instead: https://www.lexico.com/en/definition/journal.

Reply:
Done.